### ISSUE TITLE
Playwright 1.42.0 test.describe with new introduced tags syntax Second argument must be function(playwright/valid-describe-callback)

### ISSUE THREAD

**Issue by pascalknupper** [playwright-community/eslint-plugin-playwright#266]
Rule playwright/valid-describe-callback failing on new Playwright tag syntax:

ESLint: Second argument must be function(playwright/valid-describe-callback)

```typescript
test.describe('group', { tag: 'tag1' }, () => {
  test('1.1', async ({ page }) => {
    // ...
  });

  test('1.2', { tag: ['tag1.2'] }, async ({ page }) => {
    // ...
  });
});
```
